src/modules/utility-service/UtilityServiceController.ts

```diff
@@ -1,12 +1,115 @@
 import { Router } from "express";
+import proxy = require("express-http-proxy");
 import { Controller } from "../../interfaces/Controller";
+import { UTILITY_ROUTES } from "../../routes";
+
+const UTILITY_API = process.env.UTILITY_URI || 'localhost:9000';
 
 export class UtilityServiceController implements Controller {
     buildRouter(): Router {
         const router = Router();
+        
+        /**
+         * @swagger
+         * /status:
+         *  get:
+         *      description: Gets the status for the banner
+         *      tags:
+         *          - Utility Service
+         *      responses:
+         *          200:
+         *              description: OK - Returns status report
+         */
+        router.get(
+            '/status',
+            proxy(UTILITY_API, {
+                proxyReqPathResolver: req => {
+                    return UTILITY_ROUTES.STATUS;
+                },
+            }),
```

Comment:
Let's try to follow the pattern set up in the other controllers where we pass in a callback to the proxy request function.